apps/testingaccessibility/README.md

```diff
@@ -189,17 +191,71 @@ The `dev:stripe` node script is a shorthand for `stripe listen --forward-to loca
 ### Connect to the Planetscale database
 
 ```bash
-pscale connect testing-accessibility next-steps --port 3309
+pscale connect testing-accessibility PLANETSCALE_DB_BRANCH_NAME --port 3309
+```
+
+### Changing the Database Schema
+
+**Before making any changes to the schema, consider the impact those changes will have on a production system with preexisting live data.** You will likely need to take a multi-step approach to get to your desired schema. [Prisma's Expand-and-Contract Pattern guide](https://www.prisma.io/dataguide/types/relational/expand-and-contract-pattern) is a good reference for what this multi-step approach might look like.
+
+To alter the schema, you'll need to first update `/packages/database/prisma/schema.prisma`.
+
+Worth noting if you're familiar with writing migrations for database updates: With Planetscale, you alter the schema directly instead of writing migrations. The upside of this approach is that it leaves the responsibility of safe migrations up to Planetscale. The downside is that it will frequently require multiple separate releases if following the Expand-and-Contract Pattern.
+
+#### Migrate Schema Changes Locally
+
+When you make changes to the schema via Prisma you'll need to explicitly push it to the DB. We don't use Prisma migrations.
+
+You can test out your schema changes locally with the MySQL in Docker instance.
+
+First, make sure Docker is running with MySQL at the appropriate port (`3309`).
+
+Then, apply the changes to the MySQL databse using Prisma's `db push` command.
+
+```bash
+npx prisma db push
+```
+
+From here, you can connect to the MySQL database with a SQL client to view the changes or view it from _Prisma Studio_.
+
+#### Develop against the New Schema
+
+To develop against the new schema with the full power of Prisma's TypeScript typings, you'll need to re-generate the Prisma client for the updated schema. When doing this, if you switch partner products, the Prisma client will be pointing at the last partner product database until the client is regenerated.
+
+```bash
+npx prisma generate
```

Comment:
I think this is where the issue is arising from

`*prisma*` is **hoisted**

https://github.com/skillrecordings/products/blob/092366b3e24f8a002b0a86d447bf864f64941f32/.npmrc#L2

https://pnpm.io/npmrc#public-hoist-pattern

🚨 I do not fully understand public hoist pattern mechanism

So this needs to be `pnpm -- turbo run generate` from the **root of the monorepo** to generate the client that can be used by all applications.

It is **not required** to run `npx prisma generate` from within specific apps, *though it will probably work* since it has an indicator to a schema in `package.json`

`npx primsa generate` is **problematic** since it uses the latest prisma, and might conflict.

the above describes my workflow that I've used switching between apps without issue